### DISCUSSION TITLE
InfluxDB v2 Support

### DISCUSSION THREAD

**Discussion by potomacenigma** [hassio-addons/app-glances#207] [Ideas]
I guess feature requests go here? If not please tell me where? The contributing readme seems to indicate that issues were the right place, so I'm sorry for my ignorance of the correct forum.

Addon doesn't currently support InfluxDB v2 configuration parameters for Glances per:
https://glances.readthedocs.io/en/latest/gw/influxdb.html?highlight=influxdb

Can this be added to the configuration?

**Comment by atpage** [comment]
Looking for this as well.  Any way to finish/merge this?: https://github.com/hassio-addons/addon-glances/pull/217
